researcher_ids/RI_Claims_V1.md

```diff
@@ -1068,14 +1077,16 @@ the following mechanisms are available within Passport Visa:
     Clearinghouses to limit the life of access and refresh tokens.
```

Comment:
I believe it is the Broker not Claims Clearinghouse that limit the life of access and refresh tokens.

Reply:
Actually it can be either. Updated to include both and remove the refresh stuff to just state "access".